src/components/SidePanelV2/SidePanelV2.js

```diff
@@ -0,0 +1,207 @@
+import "./SidePanelV2.scss";
+import { React, useState } from "react";
+import { MdKeyboardArrowLeft, MdKeyboardArrowRight } from "react-icons/md"
+import returnToSpace from '../../assets/emojione-v1_milky-way.svg'
+import planetIcon from '../../assets/HighlightedPlanet.svg'
+import starIcon from '../../assets/Star.svg'
+import Carousel from 'react-multi-carousel'
+import 'react-multi-carousel/lib/styles.css'
+import planetdata from '../../components/Scene/ExoplanetHelper.js'
+
+const defaultValue = {
+  name: "",
+  disposition: "",
+  rightAscension: "",
+  declination: "",
+  distance: "",
+  starData: {
+    stellarDistance: "",
+    effectiveTemperature: "",
+    log: "",
+    radius: "",
+  },
+};
+
+function SidePanel({ planet }) {
+  const { name, disposition, rightAscension, declination, distance, starData } =
```

Comment:
- Instead of having planet and carouselPlanet as separate objects, you should keep track of both using one piece of state: const [planet, setPlanet] = useState(planet);
- Inside of handleCarouselPlanetClick you can set the new planet data like so: setPlanet(newPlanet);
- Then, within the side panel component, you can access the current state using object notation like so: planet.distance, planet.rightAscension, etc.  

I hope that makes sense!